assets/js/player.js

```diff
@@ -547,3 +547,14 @@ window.addEventListener('keydown', e => {
 if (player.share) {
     player.share(shareOptions);
 }
+
+// Safari audio double duration fix
+player.on('loadedmetadata', function () {
+    if (navigator.vendor == "Apple Computer, Inc." && video_data.params.listen) {
```

Comment:
Can you put the navigator.vendor check on the most outside, so that it won't add the loadedmetadata listener unless it's apple?